README.md

```diff
@@ -38,6 +38,15 @@ NOTE: If you don't have `$GOPATH/bin` in your `$PATH` you need to add it or move
 ```
 sudo mv $GOPATH/bin/rosa /usr/local/bin
 ```
+## Try the ROSA cli from binary
+
+If you don't want to build from sources you can retrieve the `rosa` binary from the latest image.
+
+You can copy it to your local with this command:
+
+```
+podman run --rm registry.ci.openshift.org/ci/rosa-aws-cli:latest cat /usr/bin/rosa > ~/rosa && chmod +x ~/rosa
```

Comment:
How does a user get access to authenticate into this registry?

Reply:
Then how can I fix this error so that I can test it?
```
$ podman run --rm registry.ci.openshift.org/ci/rosa-aws-cli:latest cat /usr/bin/rosa > ~/rosa && chmod +x ~/rosa
Trying to pull registry.ci.openshift.org/ci/rosa-aws-cli:latest...
Error: initializing source docker://registry.ci.openshift.org/ci/rosa-aws-cli:latest: unable to retrieve auth token: invalid username/password: authentication required
```

Reply:
Maybe an expired token? You can just logout and try again.

```shell
docker logout registry.ci.openshift.org
```

Reply:
This directly contradicts the statment of:
> no need. it's public

```
$ podman logout registry.ci.openshift.org
Error: not logged into registry.ci.openshift.org
```
Maybe this question still needs an answer:
> How does a user get access to authenticate into this registry?

And let's make sure it's part of the README, otherwise it will cause confusion and possibly tickets.

Reply:
It is related to openshift-ci registry way of login. Some images may need pull authentication and other may not depending on the configuration. It is documented [here](https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/#why-i-am-getting-an-authentication-error) but I'll try to add a brief explanation to the doc.

Reply:
Basicall I had to do the following and leave the password field empty:
```
$ oc registry login --registry registry.ci.openshift.org
Saved credentials for registry.ci.openshift.org
$ podman login registry.ci.openshift.org
Authenticating with existing credentials for registry.ci.openshift.org
Existing credentials are invalid, please enter valid username and password
Username (user): $USER
Password: <empty>
Login Succeeded!
```
Very confusing :/

Reply:
Hope new comments in commit help.

Reply:
Yep! I like this, thanks :)